packages/react-art/src/ReactART.js

```diff
@@ -131,6 +133,13 @@ class Text extends React.Component {
   }
 }
 
+ARTRenderer.injectIntoDevTools({
+  findFiberByHostInstance: ReactARTComponentTree.getClosestInstanceFromNode,
```

Comment:
Does this work better than just `() => null`?

My impression is we only use this feature in DevTools to highlight components from DOM nodes. But ART's host instances are not DOM nodes. So we'd never make use of this.

Am I wrong? Is there any situation where ART actually *does* use DOM nodes (e.g. SVG mode?) In that case, can we make this code work by actually finding the fiber for an SVG node?

Reply:
@gaearon Thanks for your review. If an user sets art mode to SVG, React ART renderer creates DOM nodes for SVG. So I think that 'ref' attribute of React ART's component(e.g. Surface, Group) needs to be presented on react-devtools for debugging. The lines that you said above exists for this.

Reply:
@gaearon I was a little misunderstanding about code line above . As you said, any `instance` from ReactART is not DOM node.  I will remove `ReactARTComponentTree.js` file and change this line to `() => null`

Reply:
@gaearon  I was a little misunderstanding about code line above . As you said, any instance from ReactART is not DOM node. I will remove ReactARTComponentTree.js file and change the line to findFiberByHostInstance: () => null.